2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration fullykiosk which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration browser_mod which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration ai_thermostat which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration ovapi which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration knmi which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration climate_group which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration readme which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration grocy which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration buienalarm which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration breaking_changes which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration tgtg which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration monitor_docker which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration adaptive_lighting which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration neerslag which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 14:45:55 WARNING (MainThread) [homeassistant.components.octoprint] The 'octoprint' option near /config/configuration.yaml:14 is deprecated, please remove it from your configuration
2021-11-18 14:45:56 WARNING (MainThread) [homeassistant.components.ssdp] Failed to setup listener for 0.0.0.0: [Errno 98] Address in use
2021-11-18 14:46:04 ERROR (MainThread) [homeassistant.components.weather] The buienradar platform for the weather integration does not support platform setup. Please remove it from your config.
2021-11-18 14:46:15 WARNING (MainThread) [homeassistant.components.sensor] Setup of sensor platform command_line is taking over 10 seconds.
2021-11-18 14:46:21 ERROR (SyncWorker_6) [homeassistant.components.command_line] Timeout for command: /config/scripts/get_recipies.py
2021-11-18 14:46:21 WARNING (SyncWorker_6) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:46:22 ERROR (SyncWorker_1) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:46:22 WARNING (SyncWorker_1) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'float' got invalid input 'unknown' when rendering template '{{ states('sensor.curtains_front')|float > 0 }}' but no default was specified. Currently 'float' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'float' got invalid input 'unknown' when rendering template '{{ states('sensor.curtains_front')|float > 0 }}' but no default was specified. Currently 'float' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 398, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1698, in _render_with_context
    return template.render(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.9/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1724, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1715, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
jinja2.exceptions.UndefinedError: None has no element 16

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 514, in async_render_to_info
    render_info._result = self.async_render(variables, strict=strict, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 400, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: None has no element 16
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 398, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1698, in _render_with_context
    return template.render(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.9/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1724, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1715, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
jinja2.exceptions.UndefinedError: None has no element 16

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 514, in async_render_to_info
    render_info._result = self.async_render(variables, strict=strict, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 400, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: None has no element 16
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 398, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1698, in _render_with_context
    return template.render(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.9/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1724, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1715, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
jinja2.exceptions.UndefinedError: None has no element 16

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 514, in async_render_to_info
    render_info._result = self.async_render(variables, strict=strict, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 400, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: None has no element 16
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 398, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1698, in _render_with_context
    return template.render(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.9/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1724, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1715, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
jinja2.exceptions.UndefinedError: None has no element 16

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 514, in async_render_to_info
    render_info._result = self.async_render(variables, strict=strict, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 400, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: None has no element 16
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 398, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1698, in _render_with_context
    return template.render(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.9/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1724, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1715, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
jinja2.exceptions.UndefinedError: None has no element 16

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 514, in async_render_to_info
    render_info._result = self.async_render(variables, strict=strict, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 400, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: None has no element 16
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 398, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1698, in _render_with_context
    return template.render(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.9/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1724, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1715, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
jinja2.exceptions.UndefinedError: None has no element 16

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 514, in async_render_to_info
    render_info._result = self.async_render(variables, strict=strict, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 400, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: None has no element 16
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 398, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1698, in _render_with_context
    return template.render(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.9/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1724, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1715, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
jinja2.exceptions.UndefinedError: None has no element 16

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 514, in async_render_to_info
    render_info._result = self.async_render(variables, strict=strict, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 400, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: None has no element 16
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 398, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1698, in _render_with_context
    return template.render(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.9/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1724, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1715, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
jinja2.exceptions.UndefinedError: None has no element 16

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 514, in async_render_to_info
    render_info._result = self.async_render(variables, strict=strict, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 400, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: None has no element 16
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}")' for attribute '_attr_native_value' in entity 'sensor.cooking'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}")' for attribute 'recipe_id' in entity 'sensor.cooking'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_1' in entity 'sensor.cooking'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_2' in entity 'sensor.cooking'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_3' in entity 'sensor.cooking'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_4' in entity 'sensor.cooking'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_5' in entity 'sensor.cooking'
2021-11-18 14:46:22 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_6' in entity 'sensor.cooking'
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:46:22 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:47:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:47:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:47:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:47:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:47:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:47:21 ERROR (SyncWorker_6) [homeassistant.components.command_line] Timeout for command: /config/scripts/get_recipies.py
2021-11-18 14:47:21 WARNING (SyncWorker_6) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:47:22 ERROR (SyncWorker_1) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:47:22 WARNING (SyncWorker_1) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:47:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 14:47:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 14:47:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:47:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:47:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:47:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:47:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:47:22 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:48:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:48:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:48:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:48:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:48:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:48:20 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:48:20 ERROR (SyncWorker_2) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:48:20 WARNING (SyncWorker_2) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:49:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:49:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:49:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:49:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:49:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:49:19 ERROR (SyncWorker_3) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:49:19 WARNING (SyncWorker_3) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:49:20 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:50:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:50:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:50:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:50:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:50:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:50:20 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:50:21 ERROR (SyncWorker_9) [homeassistant.components.command_line] Timeout for command: /config/scripts/get_recipies.py
2021-11-18 14:50:21 WARNING (SyncWorker_9) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list") }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}")' for attribute '_attr_native_value' in entity 'sensor.cooking'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}")' for attribute 'recipe_id' in entity 'sensor.cooking'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_1' in entity 'sensor.cooking'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_2' in entity 'sensor.cooking'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_3' in entity 'sensor.cooking'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_4' in entity 'sensor.cooking'
2021-11-18 14:50:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_5' in entity 'sensor.cooking'
2021-11-18 14:50:22 ERROR (SyncWorker_7) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:50:22 WARNING (SyncWorker_7) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:51:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:51:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:51:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:51:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:51:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:51:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:51:21 ERROR (SyncWorker_5) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:51:21 WARNING (SyncWorker_5) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:52:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:52:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:52:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:52:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:52:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:52:19 ERROR (SyncWorker_8) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:52:19 WARNING (SyncWorker_8) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:52:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:53:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:53:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:53:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:53:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:53:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:53:21 ERROR (SyncWorker_6) [homeassistant.components.command_line] Timeout for command: /config/scripts/get_recipies.py
2021-11-18 14:53:21 WARNING (SyncWorker_6) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}")' for attribute '_attr_native_value' in entity 'sensor.cooking'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}")' for attribute 'recipe_id' in entity 'sensor.cooking'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_1' in entity 'sensor.cooking'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_2' in entity 'sensor.cooking'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_3' in entity 'sensor.cooking'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_4' in entity 'sensor.cooking'
2021-11-18 14:53:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_5' in entity 'sensor.cooking'
2021-11-18 14:53:22 ERROR (SyncWorker_10) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:53:22 WARNING (SyncWorker_10) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:54:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:54:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:54:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:54:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:54:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:54:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:54:19 ERROR (SyncWorker_5) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:54:19 WARNING (SyncWorker_5) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:55:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:55:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:55:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:55:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:55:14 WARNING (Thread-110) [homeassistant.helpers.template] Template variable warning: str object has no element () when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[] }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:14 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: str object has no element () when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[] }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:14 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: str object has no element () when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[] }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:55:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:55:20 ERROR (SyncWorker_9) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:55:20 WARNING (SyncWorker_9) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:55:23 WARNING (Thread-112) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'nam' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].nam }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:23 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'nam' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].nam }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:23 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'nam' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].nam }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:24 WARNING (Thread-113) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:24 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:24 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}
{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:28 WARNING (Thread-114) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:28 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:28 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:31 ERROR (Thread-115) [homeassistant.helpers.template] Template variable error: str object has no element () when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:32 ERROR (Thread-116) [homeassistant.helpers.template] Template variable error: str object has no element 1 when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[1].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:34 ERROR (Thread-117) [homeassistant.helpers.template] Template variable error: str object has no element () when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:35 WARNING (Thread-118) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:35 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:35 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:57 WARNING (Thread-119) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:57 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:55:57 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'name' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:56:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:56:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:56:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:56:02 ERROR (Thread-120) [homeassistant.helpers.template] Template variable error: 'str object' has no attribute 'product' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:07 WARNING (Thread-121) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'product' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].product }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:07 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'product' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].product }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:07 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'str object' has no attribute 'product' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}
{{ states("sensor.grocy_shopping_list")[0].product }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:56:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:56:21 ERROR (SyncWorker_5) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:56:21 WARNING (SyncWorker_5) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:56:34 ERROR (Thread-125) [homeassistant.helpers.template] Template variable error: 'state' is undefined when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state("sensor.grocy_shopping_list")[0] }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:36 ERROR (Thread-126) [homeassistant.helpers.template] Template variable error: 'state_atrr' is undefined when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_atrr("sensor.grocy_shopping_list")[0] }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:39 ERROR (Thread-127) [homeassistant.helpers.template] Template variable error: 'state_atr' is undefined when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_atr("sensor.grocy_shopping_list")[0] }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:46 WARNING (Thread-130) [homeassistant.helpers.template] Template variable warning: None has no element 0 when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "")[0] }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:46 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: None has no element 0 when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "")[0] }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:46 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: None has no element 0 when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "")[0] }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:57 WARNING (Thread-133) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'n' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "products")[0].product.n }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:57 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'n' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "products")[0].product.n }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:56:57 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'n' when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "products")[0].product.n }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:57:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:57:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:57:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:57:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:57:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:57:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:57:21 ERROR (SyncWorker_5) [homeassistant.components.command_line] Timeout for command: /config/scripts/get_recipies.py
2021-11-18 14:57:21 WARNING (SyncWorker_5) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "products")[0].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}")' for attribute '_attr_native_value' in entity 'sensor.cooking'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}")' for attribute 'recipe_id' in entity 'sensor.cooking'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_1' in entity 'sensor.cooking'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_2' in entity 'sensor.cooking'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_3' in entity 'sensor.cooking'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_4' in entity 'sensor.cooking'
2021-11-18 14:57:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_5' in entity 'sensor.cooking'
2021-11-18 14:57:22 ERROR (SyncWorker_9) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:57:22 WARNING (SyncWorker_9) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:57:39 ERROR (Thread-135) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "products")[].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:57:40 ERROR (Thread-136) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr("sensor.recipies", "recepten")[16].photo }}

{{ state_attr("sensor.grocy_shopping_list", "products")[1].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:57:50 ERROR (Thread-137) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:57:51 ERROR (Thread-138) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:57:53 ERROR (Thread-139) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:57:59 ERROR (Thread-140) [homeassistant.helpers.template] Template variable error: list object has no element () when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:58:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:58:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:58:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:58:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:58:02 ERROR (Thread-142) [homeassistant.helpers.template] Template variable error: list object has no element () when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[].product.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:58:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:58:18 WARNING (Thread-145) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'prod' when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].prod }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:58:18 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'prod' when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].prod }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:58:18 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'prod' when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].prod }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:58:20 WARNING (Thread-146) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'products' when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].products }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:58:20 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'products' when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].products }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:58:20 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'dict object' has no attribute 'products' when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].products }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:58:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 14:58:21 ERROR (SyncWorker_9) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:58:21 WARNING (SyncWorker_9) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:58:22 ERROR (Thread-147) [homeassistant.helpers.template] Template variable error: 'dict object' has no attribute 'products' when rendering '{{ state_attr("sensor.grocy_shopping_list", "products")[1].products.name }}

{{ states("binary_sensor.wallpanel") }}
{{ state_attr('calendar.feyenoord','start_time') }}'
2021-11-18 14:59:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:59:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:59:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:59:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 14:59:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 14:59:19 ERROR (SyncWorker_7) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 14:59:19 WARNING (SyncWorker_7) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 14:59:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:00:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:00:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:00:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:00:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:21 ERROR (SyncWorker_6) [homeassistant.components.command_line] Timeout for command: /config/scripts/get_recipies.py
2021-11-18 15:00:21 WARNING (SyncWorker_6) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 16 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}")' for attribute '_attr_native_value' in entity 'sensor.cooking'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}")' for attribute 'recipe_id' in entity 'sensor.cooking'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_1' in entity 'sensor.cooking'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_2' in entity 'sensor.cooking'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_3' in entity 'sensor.cooking'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_4' in entity 'sensor.cooking'
2021-11-18 15:00:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 16') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}")' for attribute 'step_5' in entity 'sensor.cooking'
2021-11-18 15:00:22 ERROR (SyncWorker_1) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:00:22 WARNING (SyncWorker_1) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:00:56 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 15:00:56 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 15:00:56 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:56 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:56 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:56 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:56 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:00:56 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:01:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:01:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:01:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:01:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:01:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:01:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:01:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:01:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:01:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:01:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:01:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:01:19 ERROR (SyncWorker_9) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:01:19 WARNING (SyncWorker_9) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:02:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:02:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:02:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:02:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:02:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:02:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:02:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:02:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:02:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:02:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:02:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:02:19 ERROR (SyncWorker_2) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:02:19 WARNING (SyncWorker_2) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:03:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:03:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:03:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:03:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:03:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:03:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:21 ERROR (SyncWorker_10) [homeassistant.components.command_line] Timeout for command: /config/scripts/get_recipies.py
2021-11-18 15:03:21 WARNING (SyncWorker_10) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 3') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}")' for attribute '_attr_native_value' in entity 'sensor.cooking'
2021-11-18 15:03:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 3') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}")' for attribute 'recipe_id' in entity 'sensor.cooking'
2021-11-18 15:03:22 ERROR (SyncWorker_7) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:03:22 WARNING (SyncWorker_7) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:04:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:04:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:04:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:04:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:04:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:04:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:04:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:04:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:04:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:04:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:04:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:04:21 ERROR (SyncWorker_10) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:04:21 WARNING (SyncWorker_10) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:05:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:05:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:05:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:05:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:05:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:05:21 ERROR (SyncWorker_6) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:05:21 WARNING (SyncWorker_6) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:05:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:05:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:05:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:05:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:05:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:05:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:06:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:06:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:06:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:06:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:06:15 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities light.spot
2021-11-18 15:06:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:06:20 ERROR (SyncWorker_10) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:06:20 WARNING (SyncWorker_10) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:06:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:06:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:06:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:06:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:06:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:06:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:07:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:07:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:07:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:07:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (SyncWorker_6) [homeassistant.components.command_line] Timeout for command: /config/scripts/get_recipies.py
2021-11-18 15:07:21 WARNING (SyncWorker_6) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: None has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 3') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].name }}")' for attribute '_attr_native_value' in entity 'sensor.cooking'
2021-11-18 15:07:21 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: None has no element 3') while processing template 'Template("{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].id}}")' for attribute 'recipe_id' in entity 'sensor.cooking'
2021-11-18 15:07:22 ERROR (SyncWorker_5) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:07:22 WARNING (SyncWorker_5) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:08:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:08:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:08:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:08:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:08:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:08:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:08:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:08:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:08:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:08:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:08:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:08:20 ERROR (SyncWorker_3) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:08:20 WARNING (SyncWorker_3) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:09:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:09:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.lastcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:09:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:09:00 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input 'None' when rendering template '{{ as_timestamp(now()) | timestamp_custom("%H:%M") == (as_timestamp(state_attr('calendar.feyenoord', 'start_time')) |int - (state_attr('input_datetime.firstcall','timestamp'))) | timestamp_custom("%H:%M") }}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 15:09:16 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.recipies is taking over 10 seconds
2021-11-18 15:09:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 1 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[1].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:09:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 2 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[2].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:09:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 3 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[3].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:09:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 4 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[4].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:09:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 5 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[5].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:09:19 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: list object has no element 6 when rendering '{{ state_attr('sensor.recipies', 'recepten')[states('input_number.recipe_id')|round].description.split('<li>')[6].split('<br />')[0]|replace('</li>','')}}'
2021-11-18 15:09:21 ERROR (SyncWorker_2) [homeassistant.components.command_line] Command failed: /config/scripts/shopping_items.py
2021-11-18 15:09:21 WARNING (SyncWorker_2) [homeassistant.components.command_line.sensor] Empty reply found when expecting JSON data
2021-11-18 15:09:30 WARNING (MainThread) [homeassistant.components.octoprint] The 'octoprint' option is deprecated, please remove it from your configuration
2021-11-18 15:09:34 WARNING (MainThread) [homeassistant.components.octoprint] The 'octoprint' option is deprecated, please remove it from your configuration
2021-11-18 15:09:34 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aioesphomeapi/client.py", line 167, in _on_stop
    await on_stop()
  File "/usr/local/lib/python3.9/site-packages/aioesphomeapi/reconnect_logic.py", line 84, in _on_disconnect
    await self._on_disconnect_cb()
  File "/usr/src/homeassistant/homeassistant/components/esphome/__init__.py", line 291, in on_disconnect
    disconnect_cb()
  File "/usr/src/homeassistant/homeassistant/helpers/event.py", line 289, in remove_listener
    _async_remove_indexed_listeners(
  File "/usr/src/homeassistant/homeassistant/helpers/event.py", line 317, in _async_remove_indexed_listeners
    callbacks[storage_key].remove(job)
KeyError: 'light.hal'
